style.css

```diff
@@ -74,6 +73,59 @@
   position: absolute;
 }
 
+.shadow {
+   box-shadow: 0 -1px 20px 0px rgba(0, 0, 0, 0.3);
+}
+
+.footer {
+  background-color: #F5C85F;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  flex-direction: column;
+  padding: 30px;
+  padding-bottom: 50px;
+  margin: 6px 0px 0px 0px;
```

Comment:
since the margin values for this element default to 0 I would change this to: `margin-top: 6px;`